targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F415xG/objects.h

```diff
@@ -0,0 +1,53 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2017 ARM Limited
```

Comment:
Can you fix new files - add SPDX identifier ?

Reply:
See travis job license_check for details

Reply:
@0xc0170 these files are copies from other places in the codebase and tweaked - I can't relicense them as I'm not th original author.

Reply:
Understood, not all files have been fixed. It is not to relicense, just adding SPDX identifier for what is already there (the license header is present).

In this file, it would be: `SPDX-License-Identifier: Apache-2.0`

Reply:
Oh, okay. Thanks. Will do.